### DISCUSSION TITLE
You shouldn't be able to open skin layout editor during gameplay.

### DISCUSSION THREAD

**Discussion by DarkbitNetwork** [ppy/osu#30462] [Ideas]
Somebody I know on Discord accidentally opened skin layout editor while playing osu!(lazer) and failed a play because of it. He uses SD as his tapping keys and his pinky fingers press the Ctrl and Shift buttons without realizing it I guess?

Personally, I agree that skin layout editor should be disabled while playing the game (without auto, because using auto is basically watching a replay anyway) because there are no reasons to ever use skin layout editor while playing at the same time anyway.

**Comment by bdach** [comment]
You can unbind the key via settings and access the skin layout editor via settings instead.

**Comment by shiumano** [comment]
Huh? During gameplay (except during intros and breaks) you cannot open anything that takes you away from gameplay.
Settings, beat map list, chat and notifications will not open.
However, the skin layout editor is always open ...... This is a problem.

**Reply by bdach** [reply]
I guess we could do that and have the skin editor overlay respect the same behaviour the other online overlays do? It's a bit special, which is why it's currently not doing that.

**Reply by N0Y0U2114** [reply]
I think it is nice to be able to open it in gameplay as when working on a layout its nice to be able to change things and go back to playing on the fly. Maybe a toggle simillar to the windows key toggle
